### PR TITLE
sends birthdate to customer.io as timestamp instead of formatted date

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
- Customer.io wants the user's birthdate sent as a timestamp, not formatted date. This PR updates the logic to do that! 

#### How should this be reviewed?
👀 

#### Relevant Tickets
Fixes [this](https://www.pivotaltracker.com/n/projects/2019429/stories/156696372) Pivotal Card.

#### Checklist
- [ ] Documentation added for changed endpoints.
- [ ] Tests added for new features/bug fixes.
- [ ] Post a message in #api if this includes something that causes a rebuild!

### CODE REVIEW

**Comment by chloealee** [comment]
@katiecrane in the [Pivotal Card](https://www.pivotaltracker.com/n/projects/2019429/stories/156696372), the third task says that we need to update the [BackfillCustomerIo](https://github.com/DoSomething/northstar/blob/master/app/Console/Commands/BackfillCustomerIo.php) job but I don't think we need to since it sends through [`toCustomerIoPayload`](https://github.com/DoSomething/northstar/pull/736/files#diff-4614b2d052a2cb7486640775e2bdbb55L415) updated in this PR anyway?

**Comment by katiecrane** [comment]
@chloealee I think we want to update it to remove [`$query = User::where('cio_full_backfill', '!=', true);
`](https://github.com/DoSomething/northstar/blob/master/app/Console/Commands/BackfillCustomerIo.php#L34) because we want to make sure to re-send everyone

File: app/Console/Commands/BackfillCustomerIo.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ class BackfillCustomerIo extends Command
     public function handle()
     {
         // @TODO: Running this command will cause users to be sent to customer.io twice
```
Comment (DFurnes) [inline app/Console/Commands/BackfillCustomerIo.php]: This `TODO` can be removed!

Comment (chloealee) [reply app/Console/Commands/BackfillCustomerIo.php]: oh right!

**Pull request merged**